content/guides/models/track/runs/multiple-runs-per-process.md

```diff
@@ -0,0 +1,168 @@
+
+To achieve this, use `reinit="create_new"` and initialize multiple runs. For this example, suppose "Run A" is the primary process that remains open throughout the script, while "Run B1", "Run B2", are short-lived secondary runs for tasks like evaluation. 
+
+The high level workflow might look like this:
+
+1. Initialize the primary process Run A with `wandb.init()` and log training metrics.  
+2. Initialize Run B1 (with `wandb.init()`), log data, then finish it.  
+3. Log more data to Run A.  
+4. Initialize Run B2, log data, then finish it.  
+5. Continue logging to Run A.  
+6. Finally finish Run A at the end.
+
+The following Python code example demonstrates this workflow:
+
+```python
+import wandb
+
+def train(name: str) -> None:
+    """Perform one training iteration in its own W&B run.
+
+    Using a 'with wandb.init()' block with `reinit="create_new"` ensures that
+    this training sub-run can be created even if another run (like our primary
+    tracking run) is already active.
+    """
+    with wandb.init(
+        project="my_project",
+        name=name,
+        reinit="create_new"
+    ) as run:
+        # In a real script, you'd run your training steps inside this block.
+        run.log({"train_loss": 0.42})  # Replace with your real metric(s)
+
+def evaluate_loss_accuracy() -> (float, float):
+    """Returns the current model's loss and accuracy.
+    
+    Replace this placeholder with your real evaluation logic.
+    """
+    return 0.27, 0.91  # Example metric values
+
+# Create a 'primary' run that remains active throughout multiple train/eval steps.
+with wandb.init(
+    project="my_project",
+    name="tracking_run",
+    reinit="create_new"
+) as tracking_run:
+    # 1) Train once under a sub-run named 'training_1'
+    train("training_1")
+    loss, accuracy = evaluate_loss_accuracy()
+    tracking_run.log({"eval_loss": loss, "eval_accuracy": accuracy})
+
+    # 2) Train again under a sub-run named 'training_2'
+    train("training_2")
+    loss, accuracy = evaluate_loss_accuracy()
+    tracking_run.log({"eval_loss": loss, "eval_accuracy": accuracy})
+    
+    # The 'tracking_run' finishes automatically when this 'with' block ends.
+```
+
+Note three key points from the previous example:
+
+1. `reinit="create_new"` creates a new run each time you call `wandb.init()`.
+2. You keep references of each run. `wandb.run` does not automatically point to the new run created with `reinit="create_new"`. Store new runs in variables like `run_a`, `run_b1`, etc., and call `.log()` or `.finish()` on those objects as needed.
+3. You can finish sub-runs whenever you want while keeping the primary run open until.
+4. Finish your runs with `run.finish()` when you are done logging to them. This ensures that all data is uploaded and the run is properly closed.
+
+ 
+### Considerations
```

Comment:
Is this section specific to the example it is in, or did you mean for it to be an H2?

I'm not crazy about how this section looks / is structured in the output. The "if" in line 157 is doing a lot of work and could be missed. I wonder if this is actually a list of antipatterns. WDYT?